test-suites/helpers/validated-steps.ts

```diff
@@ -567,22 +595,258 @@ const checkBeforeLiquidation = async (before: LiquidationValidationData) => {
   }
 };
 
-// refactor: move check logics here before and after liquidation
-/* eslint-disable  no-unused-vars */
 const checkAfterLiquidationERC20 = async (
-  before: LiquidationValidationData,
-  after: LiquidationValidationData
+  borrower: SignerWithAddress,
+  targetToken: SupportedAsset,
+  liquidationToken: SupportedAsset,
+  amountToLiquidate: BigNumber,
+  receivePToken: boolean,
+  before: BeforeLiquidationValidationData,
+  after: AfterLiquidationValidationData
 ) => {
-  return true;
+  // borrower's Token balance is the same
+  expect(after.borrowerTargetTokenBalance).equal(
+    before.borrowerTargetTokenBalance
+  );
+
+  // borrower's pToken balance is subtracted amountToLiquidate+liq bonus (bonus is paid by borrower)
+  assertAlmostEqual(
+    after.borrowerTargetXTokenBalance,
+    before.borrowerTargetXTokenBalance.sub(
+      amountToLiquidate.mul(before.liquidationBonus).div(10000)
+    )
+  );
```

Comment:
will this assert work without consideration of price of each asset?

Reply:
yeah, I need to recheck all the logic, I put everything together so it should be simpler now. Let you know when it's done.

Reply:
I reviewed all validations, should be ok now.